scripts/get_all_heron_paths.sh

```diff
@@ -23,7 +23,19 @@ function query() {
 
 set +e
 # Build everything
-bazel build {heron,integration-test,tools/java}/...
+os_type=darwin
+case "$OSTYPE" in
```

Comment:
One issue with $OSTYPE might be the return value 'linux-gnu' for both Centos and Ubuntu and we cannot distinguish between the two.